app/controllers/shared/api.php

```diff
@@ -554,3 +554,12 @@
                 ->submit();
         }
     });
+
+App::init()
+    ->groups(['usage'])
+    ->inject('utopia')
```

Comment:
You  don't need to inject `utopia` here. You can directly use `App::getEnv()` as its a static method of the `App` class